base/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -75,14 +75,14 @@ spec:
         livenessProbe:
           httpGet:
             path: /healthz
-            port: http
+            port: debug
             scheme: HTTP
           initialDelaySeconds: 300
           timeoutSeconds: 5
         readinessProbe:
           httpGet:
-            path: /healthz
-            port: http
+            path: /ready
+            port: debug
```

Comment:
@daxmc99 @efritz I think the switch from `http` to `debug` means we will start sending traffic to a sourcegraph-frontend before the `http` port is ready. This is often the case, since `debug` starts up straight away, while `http` does some setup/migrations/etc before being ready.

Reply:
The ready endpoint should only report true directly before or after the HTTP port is accepting connections.

Pretty garbage endpoint otherwise.

Reply:
aah thanks for the clarification! See now that we pass in a channel to the ready endpoint.